en/mailing.texy

```diff
@@ -52,17 +59,26 @@ This behavior can be supressed by adding `FALSE` as a second parameter of the `s
 
 If a HTML e-mail has no plain-text alternative, it will be automatically generated. And if it has no subject set, it will be taken from the `<title>` element.
 
-Of course, it's possible to add attachments to the e-mail:
+
+Attachments
+===========
+
+Adding attachments to the e-mail is simple. In order to attach a file, we use [addAttachment|api:Nette\Mail\Message::addAttachment()] method:
 
 /--php
-$mail->addAttachment('example.zip');
-\--
+// attaches example.zip to the e-mail
+$mail->addAttachment('path/to/example.zip');
 
-Can e-mail sending be even easier?
```

Comment:
Tohle prostupuje celou dokumentací a trošku mi to začíná vadit, hlavně v EN podobě. Na blog fajn, do dokumentace to podle mě nepatří. Nechal bych čtenáře udělat si názor sám.